### ISSUE TITLE
Add TagBar support

### ISSUE THREAD

**Issue by DXist** [xolox/vim-session#14]
TagBar (https://github.com/majutsushi/tagbar) creates buffer __TagBar__ that also should be treated specially.

**Comment by bigunyak** [comment]
Yeah, for now combination session+tagbar is completely broken. :(

**Comment by Mte90** [comment]
check https://github.com/xolox/vim-session/pull/144
